contracts/wrapperSushi.sol

```diff
@@ -402,7 +402,7 @@ contract WrapAndUnWrapSushi is OwnableUpgradeable {
             sushiExchange.swapExactETHForTokens{value:amount}(0, addresses, address(this), 1000000000000000 );
```

Comment:
Same case here, please pass the min amount calculated from `userSlippageTolerance` as the first param in this method, because this is what will cause the function to revert in case of high slippage. Please [see](https://uniswap.org/docs/v2/smart-contracts/router02/#swapexactethfortokens)